File: core/src/test/java/net/dontdrinkandroot/wicket/css/StringCssClassTest.java
package net.dontdrinkandroot.wicket.css;

import org.junit.Assert;
import org.junit.Test;

/**
 * @author Philip Washington Sorst <philip@sorst.net>
 */
public class StringCssClassTest
{
    @Test
    public void testGetClassString()
    {
        CssClass cssClass = new StringCssClass("test");
        Assert.assertEquals("test", cssClass.getClassString());
    }
}

File: core/src/test/kotlin/net/dontdrinkandroot/wicket/css/StringCssClassTest.kt
package net.dontdrinkandroot.wicket.css

import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test

class StringCssClassTest {

    @Test
    fun testGetClassString() {
        val cssClass: CssClass = StringCssClass("test")
        Assertions.assertEquals("test", cssClass.classString)
    }
}